FirebaseCore/Sources/FIRVersion.m

```diff
@@ -28,5 +28,10 @@
 #define STR_EXPAND(x) #x
 
 NSString* FIRFirebaseVersion(void) {
+#ifdef FIREBASE_INTERNAL
```

Comment:
Is this acceptable? If not I can move this to a function in FIRCLSConstants.m

Reply:
Hmm, I'd prefer to determine if it would be useful to other teams before putting it here.  Without doing some investigation on the impact to the rest of Firebase, I'd prefer FIRCLSConstants.m.

Also, how about a less nebulous name than `INTERNAL`. How about `FIREBASE_1P`?

Reply:
Makes sense! Removed this change - and moved it to FIRCLSConstants.m.

Also renamed CRASHLYTICS_INTERNAL to CRASHLYTICS_1P!